app/scripts/controllers/paymentModal.js

```diff
@@ -160,19 +160,27 @@ angular
       }
 
       $scope.canEditPayment = (payment) =>
-        !payment.reported ||
-        payment.paymentType === 'SCHOLARSHIP' ||
-        payment.paymentType === 'TRANSFER';
+        payment.paymentType !== 'GIFT_CARD' &&
```

Comment:
These conditionals are getting complex. Could we do `if (payment.paymentType !== 'GIFT_CARD') { return false; }`?